tei/la-dama-duende.xml

```diff
@@ -3315,6 +3321,7 @@
             <l n="1650">Ropa blanca es y un papel.</l>
             <l>Veamos si el fraile es discreto.</l>
             <stage>Lee.</stage>
+            <p>En el poco tiempo que ha que vivís en esta casa, no se ha podido hacer más ropa; como se fuere haciendo se irá llevando. A lo que decís del amigo persuadido a que soy dama de don Luis, os aseguro que no solo lo soy, pero que no puedo serlo y esto dejo para la vista, que será presto. Dios os guarde.</p>
```

Comment:
Maybe someone with a better grasp at normalizing the orthography of Calderón's Spanish should check this version.